src/api-service/__app__/onefuzzlib/request.py

```diff
@@ -92,8 +92,8 @@ def not_ok(
         )
 
 
-def redirect(location: str) -> HttpResponse:
-    return HttpResponse(status_code=302, headers={"Location": location})
+def redirect(url: str) -> HttpResponse:
```

Comment:
I assume this change is just for reader clarity, since variable name previously referred the HTTP notion of `Location` header, whereas elsewhere in this PR, we are interacting with Azure APIs that use "location" to refer to an Azure region.

Reply:
Correct.  This change was clarity, based on grep to find everywhere we used `location`.